third_party/2and3/click/types.pyi

```diff
@@ -27,7 +27,11 @@ class CompositeParamType(ParamType):
 
 class Choice(ParamType):
     choices: Iterable[str]
-    def __init__(self, choices: Iterable[str]) -> None:
+    def __init__(
+        self,
+        choices: Iterable[str],
+        case_sensitive: Optional[bool],
```

Comment:
Not `Optional` (since it shouldn't be `None`), but has a default value.

```suggestion
        case_sensitive: bool = ...,
```

Reply:
Makes sense, shall rectify.